### ISSUE TITLE
Do we really need hate and vitriol on a plugin browser?

### ISSUE THREAD

**Issue by dzg** [danamag/stremio-addons-list#757]
### Addon Manifest URL

https://2ecbbd610840-trakt.baby-beamup.club/enough/manifest.json

### Addon Description

The plugin "Trakt TV" – with it's pro Palestine messaging – inspires hate and ugliness.
Look at the discussion at https://stremio-addons.netlify.app/trakt-tv
Disgusting.
Please tell the dev that this is not the place.
I will be urging my community to report all this on github as hate speech.

Thanks.

### Language of Content

None

### Choose Labels

- [ ] anime
- [ ] asian drama
- [ ] bollywood
- [ ] cartoons
- [ ] debrid support
- [ ] http streams
- [ ] live tv
- [ ] metadata
- [ ] misc
- [ ] movies
- [ ] music
- [ ] nsfw
- [ ] podcasts
- [ ] radios
- [ ] subtitles
- [ ] torrents
- [X] tv shows
- [ ] video games

**Comment by sleeyax** [comment]
No. We don't condone hate in our communities. Thus we've taken down the addon. 

I wish you wouldn't have gone down the road to mass-report our netlify site to get it taken down though. By doing so you've failed to spread a positive message and our community members will always remember your negative actions linking it to your political views, resulting in the opposite you were trying to achieve in the first place.

**Comment by dzg** [comment]
For the record ... I did first request and was met with hatred, abuse and vitriol, thus made to realize that path would lead nowhere, and the result achieved is in fact what we set out to do, so your argument doesn't really follow.

I would suggest that you advise your community members to not hurl racist epithets at someone who is trying to create a more peaceful environment, because otherwise _you_ end up spreading a negative message and get the opposite result you were trying to achieve.

Let's put things in order, shall we?
